src/v2i-hub/ImmediateForwardPlugin/CMakeLists.txt

```diff
@@ -1,19 +1,10 @@
-PROJECT ( ImmediateForwardPlugin VERSION 7.6.0 LANGUAGES CXX )
+project( ImmediateForwardPlugin VERSION 7.6.0 LANGUAGES CXX )
 
-SET (TMX_PLUGIN_NAME "Immediate Forward")
+set(TMX_PLUGIN_NAME "Immediate Forward")
 
-FIND_PACKAGE (XercesC REQUIRED)
```

Comment:
Were we using xerces-c before? I see it's still in the install and dependencies scripts.

Reply:
It is used in other Plugins. I think the CSWPlugin uses it. We should probably remove it eventually since we have other libraries that can parse xml. But this plugin does not use it.